src/components/ArticleCards.jsx

```diff
@@ -0,0 +1,32 @@
+import { useEffect, useState } from "react";
+import { fetchArticles } from '../Api'
+
+const ArticleCards = ({ setIsLoading }) => {
+    const [articles, setArticles] = useState([])
+      useEffect(() => {
+        setIsLoading(true)
+        fetchArticles()
+        .then((articles) => {
+          setArticles(articles)
+      })
+      setIsLoading(false)
+    }, [])
+  
+    return <>
+    {
+    articles.map((article) => {
+      if (article.article_id < 10) {
```

Comment:
not clear to me why the `article_id` should be < 10 to render a card.  🤷 
We'd never see any new ones.